apps/src/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -32,7 +33,7 @@ export default function SingleSectionSetUp({
         values={section.grades || []}
         setValues={g => updateSection('grades', g)}
       />
-      <hr />
+      <br />
```

Comment:
I think we generally prefer using CSS over break tags, though I don't think it's a big deal

Reply:
fair. let me see if i can figure out some css padding.